how-to/template-how-to.md

```diff
@@ -1,54 +1,48 @@
 <!-- Copy this Template. -->
-<!-- Describe the title of your article by replacing "How-to Template" with the page name you want to publish to. -->
-# How-to template
+<!-- Describe the title of your article by replacing "How To Template" with the page name you want to publish to. -->
+# How To template
 
 ## Overview
 
-Summarise what this how-to article is about in a sentence or two.
-How-to articles primarily include tasks, and answer the question "how do I do it?".
-Readers who are reading tasks are engaging in a learning behavior.
+Summarize what this How To article is about in a sentence or two.
+How To articles primarily include tasks, and answer the question "how do I do it?".
 
-Readers should already have read a concept about the topic in an About article.
-Ensure you link to the relevant About article here.
-
-
-**Keywords:** Optionally add comma-separated keywords.
+**Keywords:** Optional: add comma-separated keywords.
 
 ## Before you start
-<!-- Delete this section if your readers can dive straight into the lesson without requiring any prerequisite knowledge. -->
-Make sure you meet the following prerequisites before starting the how-to steps:
+<!-- Delete this section if your readers can go to the steps without requiring any prerequisite knowledge. -->
+Before you begin, make sure you meet these prerequisites:
 
-* Prerequisite one
-* Prerequisite two
-* etc
+* Pre-requisite one
+* Pre-requisite two
 
-Include other important information here too, such as known issues or bugs.
+Include other important information here, such as known issues or bugs.
 
 ## Step-by-step guide
 
 ### Step 1: Optional: Title for step - image or screenshot
```

Comment:
Can you even have two colons in a row like this?

Would it be better to have it like this:

`### (Optional) Step 1: Title for step - image or screenshot`

Reply:
I don't see why you couldn't have two, but I agree that the parenthesis option might look better.